genestack_client/scripts/upload_reference_genome.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3.9
```

Comment:
Why specific Python version?  Also, Python Client is still supposed to work with Python 2.7, so I don't think it's acceptable to have a Python 3-only script here

Reply:
I will comment regarding both this and in general why this script is in this repo. We need to simplify the process of reference genome upload to ODM, so I needed to introduce a reusable example of how to upload new genomes. Do you think it doesn't fit this repo? How should it be distributed to clients at all? Maybe they don't need such example?
There is a second reason I put it here - I wanted to reuse it in future to upload genomes at ODM init. Now there's a copy of this script in unified repo, and I want to eventually get rid of it in favour of import from python client

Reply:
The version is stated like this, again, because it was just copied as is from the unified repo. I will test it using python 2.7 to make sure it works for that version as well, if it should be compatible with both